KinSDK/KinSDK/Core/KinClient.swift

```diff
@@ -135,15 +135,15 @@ public final class KinClient {
     /**
      Cached minimum fee.
      */
-    private var _minFee: Stroop?
+    private var _minFee: Quark?
 
     /**
      Get the minimum fee for sending a transaction.
 
      - Returns: The minimum fee needed to send a transaction.
      */
-    public func minFee() -> Promise<Stroop> {
-        let promise = Promise<Stroop>()
+    public func minFee() -> Promise<Quark> {
```

Comment:
elaborate in method docs what's a Quark

Reply:
Quark is now what the blockchain team is calling the previously named Stroop